src/common/Icons.js

```diff
@@ -105,4 +106,5 @@ export const IconAttach: SpecificIconType = makeIcon(Feather, 'paperclip');
 export const IconAttachment: SpecificIconType = makeIcon(IoniconsIcon, 'document-attach-outline');
 export const IconGroup: SpecificIconType = makeIcon(FontAwesome, 'group');
 export const IconPlus: SpecificIconType = makeIcon(Feather, 'plus');
+// eslint-disable-next-line react/function-component-definition
```

Comment:
nit at branch level:

    I caught these while experimenting with a new(ish eslint rule:
      react/function-component-definition
      https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/function-component-definition.md

I think my ideal format for making this kind of lint-rule-directed fix is to have the same commit that fixes the things it finds also enable the rule. So e.g. the branch would first add or upgrade the plugin that makes the rule available; in the commit that does so, disable any new rules (that affect anything) in order to avoid touching code; then later commits enable the rules one at a time.

For an example of that format, you can look at the series in TsFlower where I added ESLint: the last N commits at the top of https://github.com/gnprice/tsflower/commits/2a815a21523dca1b498d47397a9cca9f0cc6bc68, in particular https://github.com/gnprice/tsflower/commit/ac22836e70d89c39f32492e78114d0b24ba8e728 which enabled it but with several rules disabled, then various commits like https://github.com/gnprice/tsflower/commit/df966a282d85e602346e1bce22b4a5a1959ddba5 removing a disable and fixing some code, or https://github.com/gnprice/tsflower/commit/2a815a21523dca1b498d47397a9cca9f0cc6bc68 leaving a rule disabled but changing the comment from saying "plan to investigate and enable" to saying "disabled because such-and-such".

A bonus is that eslint-disable lines like this appear only at the same time as they become needed (while also still appearing in the same commit that handles the other instances of the same lint rule.)

But these commits are already quite clear, so if it'd be a pain to revise the branch into that form then this is fine.